src/types/user/user-interfaces/user.interfaces.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from '~/types'
 
 export interface LocalStorage {
-  accessToken?: string
+  'emoji-mart.last'?: string
```

Comment:
what is that for?

Reply:
this interface represents the expected structure of the data stored in the local storage. 'emoji-mart.last' is the item that we store in our local storage

We use it in local-storage-service:
JSON.parse(localStorage.getItem('s2s') || 'null') as LocalStorage | null